jsx/StaticDataTable.js

```diff
@@ -91,7 +91,7 @@ class StaticDataTable extends Component {
    * @param {object} prevState - Previous React Component state
    */
   componentDidUpdate(prevProps, prevState) {
-    if (jQuery.fn.DynamicTable) {
+    if (jQuery.fn.DynamicTable && !this.props.NoDynamicTable) {
```

Comment:
can we add this new prop to StaticDataTable.propTypes and StaticDataTable.defaultProps